cmd/cainjector/app/start.go

```diff
@@ -68,7 +70,7 @@ func NewInjectorControllerOptions(out, errOut io.Writer) *InjectorControllerOpti
 }
 
 // NewCommandStartInjectorController is a CLI handler for starting cert-manager
-func NewCommandStartInjectorController(out, errOut io.Writer, stopCh <-chan struct{}) *cobra.Command {
+func NewCommandStartInjectorController(ctx context.Context, out, errOut io.Writer) *cobra.Command {
```

Comment:
I've gone through replacing `stopCh` with cancellable context so that all the go routines launched by us can be managed with `errgroup.Group`.